Quadriga/src/main/java/edu/asu/spring/quadriga/validator/ProjectValidator.java

```diff
@@ -1,112 +1,38 @@
-	@Override
-	public boolean supports(Class<?> arg0) {
-		return arg0.isAssignableFrom(Project.class);
-	}
 
-	@Override
-	public void validate(Object obj, Errors err) 
-	{
-		//validate all the input parameters
-		ValidationUtils.rejectIfEmptyOrWhitespace(err, "projectName", "project_name.required");
-		ValidationUtils.rejectIfEmptyOrWhitespace(err, "description", "project_description.required");
-		ValidationUtils.rejectIfEmptyOrWhitespace(err, "projectAccess", "project_projectAccess.required");
-		//ValidationUtils.rejectIfEmptyOrWhitespace(err, "unixName", "project_unixname.required");
-		
-		Project project = (Project)obj;
-		
-		String projUnixName = project.getUnixName();
-		String projectId = project.getProjectId();
-		EProjectAccessibility projectAccess = project.getProjectAccess();
-		
-		if(err.getFieldError("projectAccess")==null)
-		{
-			//validate the selected project accessibility
-			validateProjectAccessibility(projectAccess,err);
-		}
-		
-		if(err.getFieldError("unixName")==null) {
-    		//validate the regular expression
-    		validateUnixNameExp(projUnixName,err);
-		}
-		
-		
-		if(err.getFieldError("unixName")==null) {
-			try {
-				validateUnixName(projUnixName,projectId,err);
-			} catch(QuadrigaStorageException e) {
-				logger.error("Error", e);
-			}
-		}
-	}
-	
-	public void validateProjectAccessibility( EProjectAccessibility projectAccess,Errors err)
-	{
-		if(projectAccess == null)
-		{
-			err.rejectValue("projectAccess", "project_projectAccess_selection.required");
-		}
-		
-	}
-	
-	/**
-	 * This method validates if the Unix name contains letters other than the
-	 * mentioned value in the regular expression
-	 * @param unixName
-	 * @param err
-	 * @author kiran batna
-	 */
-	public void validateUnixNameExp(String unixName,Errors err)
-	{
-		String regex = "^[a-zA-Z0-9-_.+!*'()]*$";
-		Pattern pattern = Pattern.compile(regex);
-		
-		if(!pattern.matcher(unixName).matches())
-		{
-			err.rejectValue("unixName","project_UnixName.expression");
-		}
-	}
+    @Override
+    public boolean supports(Class<?> arg0) {
+        return arg0.isAssignableFrom(Project.class);
+    }
+
+    @Override
+    public void validate(Object obj, Errors err) {
+        // validate all the input parameters
+        ValidationUtils.rejectIfEmptyOrWhitespace(err, "projectName", "project_name.required");
+        ValidationUtils.rejectIfEmptyOrWhitespace(err, "description", "project_description.required");
+        ValidationUtils.rejectIfEmptyOrWhitespace(err, "projectAccess", "project_projectAccess.required");
+        // ValidationUtils.rejectIfEmptyOrWhitespace(err, "unixName",
+        // "project_unixname.required");
+        Project project = (Project) obj;
+        EProjectAccessibility projectAccess = project.getProjectAccess();
+        validateProjectAccessibility(projectAccess, err);
+    }
+
+    public void validateProjectAccessibility(EProjectAccessibility projectAccess, Errors err) {
+        if (projectAccess == null) {
```

Comment:
this is already tested with "rejectIfEmptyOrWhitepace" isn't it? Wasn't that what we've talked about today?